ckanext/tracking/middleware.py

```diff
@@ -35,15 +34,13 @@ def track_request(response: Response) -> Response:
         h.update(key.encode())
         key = h.hexdigest()
         # store key/data here
-        sql = '''INSERT INTO tracking_raw
-                    (user_key, url, tracking_type)
-                     VALUES (:key, :url, :type)'''
-
-        with cast(sa.engine.Engine, engine).begin() as conn:
-            conn.execute(sa.text(sql), {
-                "key": key,
-                "url": data.get("url"),
-                "type": data.get("type"),
-            })
+        try:
+            TrackingRaw.create(
+                user_key=key,
+                url=data.get("url"),
+                tracking_type=data.get("type")
+            )
+        except Exception as e:
+            raise e
```

Comment:
It's probably non required to track the exception just to raise it. 
Maybe you want to do something else (or maybe just drop the try-catch here)

Reply:
I think its good to log the error but i forgot to add it and remove the `raise e`